blueprints/addon/files/.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - [main, master]
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: 14.x
```

Comment:
This is node 14 instead of node 12.  Doesn't line up exactly with the travis file, but perhaps that is preferred by us?

Reply:
This should probably be the earliest version of node currently supported

Reply:
Thanks Katie!  Looks like that would be Node 12 then.  I put node 14 b/c it was the active version but if our earliest supported version, then I will change this

https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md#current-support
https://nodejs.org/en/about/releases/